mailpoet/lib/Config/Migrator.php

```diff
@@ -932,4 +935,29 @@ private function migrateEmailActionsFilters(): bool {
     }
     return true;
   }
+
+  private function updateDefaultInactiveSubscriberTimeRange(): bool {
+    // only run if the current database version is older than 3.78.0
+    if (version_compare((string)$this->settings->get('db_version', '3.78.0'), '3.78.0', '>=')) {
```

Comment:
This is a little bit different than how other migrations handle their version checks, such as [this recent check](https://github.com/mailpoet/mailpoet/blob/791f40fd7c2a04b8f2a2b4fc12807e12e67b56b4/mailpoet/lib/Config/Migrator.php#L865). 

The way we've been doing it seems like saying "Run the migration unless the user has a version greater than the release before the migration". My version is more like "Run the migration unless the user has a version matching or greater than the release with the migration" (assuming this migration will be released with version `3.78.0`).

With the current style, we have to update the checks for unreleased migrations if we have a patch release, like [this commit](https://github.com/mailpoet/mailpoet/commit/791f40fd7c2a04b8f2a2b4fc12807e12e67b56b4).

Is it not a safe assumption on my part that migrations for normal priority changes like this PR will go out with the next minor release version?

I feel like I'm missing something obvious.

Reply:
@lysyjan Thanks for the in depth explanation. I was making an assumption that we wouldn't merge this branch into `master` until the next minor release, but it sounds like that assumption isn't accurate, or perhaps I'm still not understanding this 100%. I probably need to review the release process. In any case, I'm happy to rewrite this to follow the pattern of our other migrations.

Reply:
@lysyjan I just wanted to double check with you -- would the migration as I originally had it be problematic if we ensured that the code was released with version `3.78.0`? And is there a reason we would have included it in a `3.77.*` release? I just want to make sure I'm understanding the process and how the issue you described could have occurred.